### DISCUSSION TITLE
Question: does anyone know how to make it so a mod has its own save data for scoring

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1522] [Q&A] [answered]
adding an extra week will not override your score. for instance, the whitty mod is technically week 7 so any score you get on there will be your score in week 7. Is there a way I can make it so my mod has its own save data that doesn't interfere?

**Comment by Injourn** [comment, marked as answer]
Look for instances of "FlxG.save.data" such as FlxG.save.data.songScores and remove or replace where you see necessary.
